build.sbt

```diff
@@ -18,6 +18,12 @@ ThisBuild / versionScheme := Some(VersionScheme.SemVerSpec)
 sourceDistName := "apache-pekko"
 sourceDistIncubating := true
 
+commands := commands.value.filterNot { command =>
+  command.nameOption.exists { name =>
+    name.contains("sonatypeRelease") || name.contains("sonatypeBundleRelease")
```

Comment:
I would like to use `StringUtil.containsAny(...)` :)

Reply:
I don't think we can use that because this is an sbt build and hence the dependency that brings in `StringUtil` isn't available

Reply:
seems better to stick with plain Java/Scala